src/data.json

```diff
@@ -165,7 +165,11 @@
             "premium plugins",
             "Kotlin",
             "Scala",
-            "Groovy"
+            "Groovy",
+            "Rift",
```

Comment:
wtf is rift?

Reply:
https://github.com/DimensionalDevelopment/Rift/

Reply:
its also dead by that link, lol